File: 1985.py
n = int(input())
soma = 0
for i in range(0, n):
    a, b = input().split()
    a = int(a)
    b = int(b)
    if a == 1001:
        c = 1.50 * b
        soma += c
    if a == 1002:
        c = 2.50 * b
        soma += c
    if a == 1003:
        c = 3.50 * b
        soma += c
    if a == 1004:
        c = 4.50 * b
        soma += c
    if a == 1005:
        c = 5.50 * b
        soma += c
print(f'{soma:.2f}')

File: 1985.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

let n = parseInt(lines.shift());
let soma = 0;
for(var i = 0; i < n; i++){
    let [a, b] = lines.shift().split(" ");
    a = parseInt(a); b = parseInt(b);
    if(a === 1001){
        c = 1.50 * b;
        soma += c;
    }
    if(a === 1002){
        c = 2.50 * b;
        soma += c;
    }
    if(a === 1003){
        c = 3.50 * b;
        soma += c;
    }
    if(a === 1004){
        c = 4.50 * b;
        soma += c;
    }
    if(a === 1005){
        c = 5.50 * b;
        soma += c;
    }
}
console.log(`${soma.toFixed(2)}`);
